files/generate_vars.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+import configparser
+import subprocess
+import shlex
+
+
+class GenerateMappingFile():
+    prefix = "[Generate Mapping File] "
+
+    def run(self):
+        print("\n%sStart Generate Varialbe Mapping File "
+              "For oVirt Ansible Disaster Recovery\n" % self.prefix)
+        site, username, password, ca, output_file = self._init_vars()
+        print("%ssite address: %s \n"
+              "%susername: %s \n"
+              "%spassword: %s \n"
```

Comment:
Please don't output the password. You can output "password: <hidden>" if you want.

Reply:
will do